src/mbgl/sprite/sprite_parser.cpp

```diff
@@ -24,7 +25,11 @@ SpriteImagePtr createSpriteImage(const PremultipliedImage& image,
         ratio <= 0 || ratio > 10 ||
         srcX >= image.size.width || srcY >= image.size.height ||
         srcX + width > image.size.width || srcY + height > image.size.height) {
-        Log::Error(Event::Sprite, "Can't create sprite with invalid metrics");
+        Log::Error(Event::Sprite, "Can't create sprite with invalid metrics: " +
```

Comment:
Use `Log` varargs support rather than string concatenation.

```
Log::Error(Event::Sprite, "Can't create sprite with invalid metrics: %dx%d@...",
           width, height, ...);
```